scripts/dispatch

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -ex
+
+REPO="https://api.github.com/repos/rancher/rancher/actions/workflows/replace-env-value.yml/dispatches"
+
+case $DRONE_TAG in
+  "v2.6"*)
```

Comment:
My only concern is that this breaks when we move off of `v2.6`.. I'm wondering if we can automatically add another tag for drone to pick up on a release.

Reply:
I wanted to control it a bit more but I guess we can make it generic, I changed it to generate the target branch from the tag that is created, see inline comments in the new code.